website/docs/d/network_security_group.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_network_security_group"
+sidebar_current: "docs-azurerm-datasource-network-security-group"
+description: |-
+  Get information about the specified Network Security Group.
+---
+
+# azurerm_network_security_group
+
+Use this data source to access the properties of a Network Security Group.
+
+## Example Usage
+
+```hcl
+data "azurerm_network_security_group" "test" {
+  name                = "${azurerm_network_security_group.test.name}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+}
+
+output "location" {
+  value = "${data.azurerm_network_security_group.test.location}"
+}
+```
+
+## Argument Reference
+
+* `name` - (Required) Specifies the Name of the Network Security Group.
+* `resource_group_name` - (Required) Specifies the Name of the Resource Group within which the Network Security Group exists
+
+
+## Attributes Reference
+
+* `id` - The ID of the Network Security Group.
+
+* `location` - The supported Azure location where the resource exists.
+
+* `security_rule` - One or more `security_rule` blocks as defined below.
+
+* `tags` - A mapping of tags to assign to the resource.
+
+
+The `security_rule` block supports:
+
+* `name` - The name of the security rule.
+
+* `description` - The description for this rule.
+
+* `protocol` - The network protocol this rule applies to.
+
+* `source_port_range` - The Source Port or Range.
+
+* `destination_port_range` - The Destination Port or Range.
+
+* `source_address_prefix` - CIDR or source IP range or * to match any IP.
+
+* `destination_address_prefix` - CIDR or destination IP range or * to match any IP.
+
+* `access` - Is network traffic is allowed or denied?
+
+* `priority` - The priority of the rule
+
+* `direction` - The direction specifies if rule will be evaluated on incoming or outgoing traffic.
```

Comment:
Nitpick, but is there any reason for all the newlines between each attribute?

Reply:
not particularly - I copied this over from the resource docs and updated the descriptions (since the schema's the same) - it's mostly just to help readability when writing docs (this doesn't impact the rendering on the website at all). I can consolidate it if you want?

Reply:
chatted offline about this, this is 👍 to merge - merging